cmd/renterd/main.go

```diff
@@ -321,8 +341,24 @@ func main() {
 		log.Println("Build Date:", builddate)
 		return
 	} else if flag.Arg(0) == "seed" {
-		log.Println("Seed phrase:")
-		fmt.Println(wallet.NewSeedPhrase())
+		var seed [32]byte
```

Comment:
I was actually going to suggest doing this by default. I think having a verbose CLI is fine but a tool that generates a seed should print 12 words and that's it. Currently we print something like:

```
renterd v0.6.0
Network Mainnet
Seed phrase:
```

I think printing the address is a nice addition though, but maybe we can print that every time `renterd` is started?
What if instead of a `--headless` flag we add a `--help` that gives some more ELI5 info about the seed and what it's used for and why you should make sure you have it backed up.

Reply:
Changing nothing for this. Kindly resolve this or let me know if any action is required

Reply:
I think we want to make the behaviour consistent with `hostd`, meaning that if the user types "seed" we'll generate one for him. Chris is suggesting that _if we do that_ (which we do) it's fine to only print the seed here, which we are currently not doing. This is important because these tools are often used for scripting and we don't want everyone to have to parse out the seed from the command's output, but instead the command should only print the seed and nothing more.